tweets/NodeCertSale2.tweet

```diff
@@ -0,0 +1 @@
+Big discounts on Node.js certifications and trainings offered by @openjsf are ending April 9. Check out the release of the new testing environment built for the Node.js Certifications, too! More details here: https://openjsf.org/blog/2021/03/29/node-js-certifications-and-training-sale/ 
```

Comment:
This is worded in a very marketing-ish way that hasn't typically resonated with the account's audience. Going to suggest a simplification/rephrasing:

```suggestion
The @openjsf is currently running a discount on their Node.js certifications. You can find details on their blog: https://openjsf.org/blog/2021/03/29/node-js-certifications-and-training-sale/ 
```